dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskExecuteResponseAckProcessor.java

```diff
@@ -58,7 +58,15 @@ public void process(Channel channel, Command command) {
         if (taskExecuteResponseAckCommand.getStatus() == ExecutionStatus.SUCCESS.getCode()) {
             ResponseCache.get().removeResponseCache(taskExecuteResponseAckCommand.getTaskInstanceId());
             TaskCallbackService.remove(taskExecuteResponseAckCommand.getTaskInstanceId());
-            logger.debug("remove REMOTE_CHANNELS, task instance id:{}", taskExecuteResponseAckCommand.getTaskInstanceId());
+            if (logger.isDebugEnabled()) {
```

Comment:
no need to judge here, when use `logger.debug`, it will check internally.

Reply:
Yes, it will check internally, but this judge can help to speed up this judge, I will revert this.